services/ui-src/src/components/statusing/StatusIcon.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { render } from "@testing-library/react";
+import { StatusIcon } from "./StatusIcon";
+// types
+import { ReportType } from "types";
+
+describe("StatusIcon functionality", () => {
+  it("should render the correct status for complete SAR rows", () => {
+    const { container } = render(
+      <StatusIcon reportType={ReportType.SAR} status={true} />
+    );
+    const errorImages = container.querySelectorAll(
+      "img[alt='Error notification']"
+    );
+    const successImages = container.querySelectorAll(
+      "img[alt='Success notification']"
+    );
+    expect(successImages).toHaveLength(1);
+    expect(successImages[0]).toBeVisible();
+    expect(errorImages).toHaveLength(0);
```

Comment:
This seems like it could produce a false positive if the image is changed but this test isn't updated. Use a constant for the `alt` value?